manifests/03_configmap.yaml

```diff
@@ -7,3 +7,14 @@ data:
   config.yaml: |
     apiVersion: operator.openshift.io/v1
     kind: GenericOperatorConfig
+
+---
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: openshift-global-ca
+  namespace: openshift-controller-manager
+  labels: 
+    config.openshift.io/inject-trusted-cabundle: "true"
+data: {}
```

Comment:
this file is not a resource list, so you can't just add another resource like this.

the service-ca CM is managed as bindata, seen here:
https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/bindata/v3.11.0/openshift-controller-manager/openshift-service-ca-cm.yaml

https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/sync_openshiftcontrollermanager_v311_00.go#L76

https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/sync_openshiftcontrollermanager_v311_00.go#L198-L228

so that's the pattern to copy.

Reply:
yeah I waffled on which way to go

I convinced myself that the service-ca stuff existed in 3.11, and that is why it made sense for it to go there

Reply:
That code isn't really related to 3.11... all those "311" named things are leftovers from when it was assumed we'd be migrating clusters from 3.11 to 4.x.  But we never used service CA injection in 3.11, we just used the CA that was auto mounted into the pods.